Formula/v/valkey.rb

```diff
@@ -1,8 +1,8 @@
 class Valkey < Formula
   desc "High-performance data structure server that primarily serves key/value workloads"
   homepage "https://valkey.io"
-  url "https://github.com/valkey-io/valkey/archive/refs/tags/redis-7.2.4.tar.gz"
```

Comment:
upstream might need to use version higher than 7.2.4 for rc release

Reply:
How did you get `redis-7.2.4` into your repo? Was it added automatically?

It's just a tag, not intended as a release. (It's a redis release with a renamed tag, kept for convenience mostly for comparing git history, etc.)